### DISCUSSION TITLE
CORTX on OVA quick guide for JP customer

### DISCUSSION THREAD

**Discussion by TaroIwata** [Seagate/cortx#466] [Show and tell] [answered]
Qiita is most famous technical blog in Japan.
I hope it makes help for JP contributor.
https://qiita.com/Taroi_Japanista/items/0ac03f55dce3f7433adf

Please read below for more detail.
https://github.com/Seagate/cortx/blob/main/doc/CORTX_on_Open_Virtual_Appliance.rst

**Comment by johnbent** [comment, marked as answer]
Excellent.  Thanks @TaroIwata !  We've updated https://github.com/Seagate/cortx/blob/main/doc/CORTX_on_Open_Virtual_Appliance.rst to add this link.
